### ISSUE TITLE
Default verb menu

### ISSUE THREAD

**Issue by ArturM** [godotengine/escoria#277]
I think default verb menu (demo/ui/verb_menu.tscn) should be 'classic' 9-verbs menu.
Something like that:

Open | Pick up | Push
Close | Look at | Pull
Give  | Talk to | Use

Fonts should be bigger and buttons should be aligned.

Action 'Give' should behave like 'Use' with 'use combine' enabled - because "Give" will always have two parameters - what_to_give and give_to_whom.

I can do it and make PR.
What do you think?

**Comment by ArturM** [comment]
In game.gd in line 255 we have:

`	if vm.current_action == "use" and obj.use_combine:
		if not vm.current_tool:
			vm.set_current_tool(obj)
		elif vm.current_tool != obj:
			interact([obj, vm.current_action, vm.current_tool])
`
it looks that changing first line to:

`	if (vm.current_action == "use" and obj.use_combine) or vm.current_action == "give":`

is all we need to make 'give' work as expected :)

**Comment by StraToN** [comment]
This topic relates to UI management. I believe Escoria doesn't give enough "freedom" to game creators yet. 

I've been doing a little bit of research some time ago about the different types of "player input" solutions in point and click games. The 9-verbs is one, but there are many popular ones such as the "left click-action, right click-action" (usually "left-use, right-look", but I've also met "left-use, right-inventory"). Both are very different as the former is graphical (the 9 verbs are displayed at all time, except cutscenes) with no configuration while the latter has no UI, but is configurable.

Escoria can (and should) be friendly to all player input methods. It will provide template scenes for the most typical graphical methods (such as 9 verbs), but backend scripts should be agnostic of the player input method.

In conclusion, the player input management is supposed to be separated in the future  Escoria Reloaded so it is made easier for the game developer what method he'll want to use in his game, what action verbs, etc. I'd try to look further if doing so could be possible in Escoria 3.0 already.

**Comment by ArturM** [comment]
In 'simpler' interfaces - 'left-use, right-look' and contextual icons there are no 'give' action or 'use' with two arguments. 'Use' in this games executes default action on hotspot. To use 'something on something' player must drag and drop item from inventory to hotspot (or drag item on item in inventory to combine items).
To 'give' something player must drag item from inventory and put it on NPC.

All actions that takes one argument are working, but 'use' and 'give' are special - 'use' may have one argument ("use lever") or two arguments ("use key on door").
'Give' is like 'use' but MUST have two arguments: one from inventory and another on screen.

I think Escoria's default 'old template' should be able to do all actions from classic games - now every action from 9-verb classics is doable except 'give'.

In 'new template' we can code drag and drop from inventory to NPC and drag and drop from inventory to inventory as :use - there's no need for separate 'give'.

**Comment by StraToN** [comment]
Escoria rewrite has been done with special attention given to this feature. Arriving soon. Thanks!
